README.md

```diff
@@ -20,6 +24,8 @@ sudo python3 -m pip install <path/to/setup.py>
 
 ### Changelog
 
+Version 0.14 - Missing PY_SSIZE_T_CLEAN macro added, fix for newer Python 3 versions
```

Comment:
I forgot to specify the Python 3 version in my PR back then. Maybe the exact Python 3 version could be specified (which makes the macro mandatory)?

Reply:
Good point, let me specify exact version - 3.10